common/attest_plugin.h

```diff
@@ -14,7 +14,7 @@
 #include <stdio.h>
 #include <string.h>
 
-#include "common.h"
```

Comment:
In `evidence.cpp`, `oe_attestation_header` is needed for dumping so `common/attest_plugin.h` is included. But "common.h" here will be conflicted with `FILE`, which is needed for binary output. And "common.h" seems not needed here. @shnwc is it fine to remove this header?
```
In file included from ../tests/tools/oecert/host/evidence.cpp:28:
In file included from ../tests/tools/oecert/host/../../../../common/attest_plugin.h:17:
In file included from ../tests/tools/oecert/host/../../../../common/common.h:33:
../tests/tools/oecert/host/../../../../common/oe_host_stdio.h:13:16: error: typedef 'FILE' cannot be referenced with a struct specifier
typedef struct FILE OE_FILE;
               ^
/usr/include/x86_64-linux-gnu/bits/types/FILE.h:7:25: note: declared here
typedef struct _IO_FILE FILE;
                        ^
1 error generated.
ninja: build stopped: subcommand failed.
```